back-end/requests/put/add_members.js

```diff
@@ -0,0 +1,43 @@
+let Playlist = require("../../models/playlists.model");
+
+const add_member = async (req, res) => {
+  const group_id = req.params.group_id
+  const user_id = req.params.user_id
+
+  let members
+  let banned_members
+  await Playlist.findOne({_id:group_id}) //retrieve the group information
```

Comment:
Good job on the error checking, but there are some issues with it. First, if you return from within this await method (and await methods in general), it won't actually return for the overall function (in this case, add_member). Instead, it will return a promise. So if you instead change this line to `let error = await Playlist.findOne({_id:group_id})`, then `error` will hold the error (or nothing if it was successful).

Another important note, we want the error to be sent back to the user, which will be handled by middleware. We can'